packages/vue/src/Form/DatePicker/DatePicker.vue

```diff
@@ -53,7 +53,7 @@ const props = defineProps({
   },
   placeholder: {
     type: String,
-    default: 'DD/MM/YYYY'
+    default: 'YYYY-MM-DD'
```

Comment:
I believe the changes will introduce breaking changes. Can props pass in parent component instead of update default value?

Reply:
This  won't be a breaking change, the placeholder prop wasn't being used in the component.